lib/constants/acqCommands.js

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import { FormattedMessage } from 'react-intl';
+
+export const acqCommands = [
+  {
+    name: 'new',
+    label: (<FormattedMessage id="stripes-acq-components.shortcut.createRecord" />),
+    shortcut: 'alt+n',
+  },
+  {
+    name: 'edit',
+    label: (<FormattedMessage id="stripes-acq-components.shortcut.editRecord" />),
+    shortcut: 'mod+alt+e',
+  },
+  {
+    name: 'save',
+    label: (<FormattedMessage id="stripes-acq-components.shortcut.saveRecord" />),
+    shortcut: 'mod+s',
+  },
+  {
+    name: 'expandCollapse',
+    label: (<FormattedMessage id="stripes-acq-components.shortcut.expandCollapse" />),
+    shortcut: 'spacebar',
+  },
+  {
+    name: 'expandAllSections',
+    label: (<FormattedMessage id="stripes-acq-components.shortcut.expandAll" />),
+    shortcut: 'mod+alt+b',
+  },
+  {
+    name: 'collapseAllSections',
+    label: (<FormattedMessage id="stripes-acq-components.shortcut.collapseAll" />),
+    shortcut: 'mod+alt+g',
+  },
+  {
+    name: 'search',
+    label: (<FormattedMessage id="stripes-acq-components.shortcut.goToSearchFilter" />),
+    shortcut: 'mod+alt+h',
+  },
+  {
+    name: 'close',
+    label: (<FormattedMessage id="stripes-acq-components.shortcut.closeModal" />),
+    shortcut: 'esc',
+  },
+  {
+    name: 'copy',
+    label: (<FormattedMessage id="stripes-acq-components.shortcut.copy" />),
+    shortcut: 'mod+c',
+  },
+  {
+    name: 'cut',
+    label: (<FormattedMessage id="stripes-acq-components.shortcut.cut" />),
+    shortcut: 'mod+x',
+  },
+  {
+    name: 'paste',
+    label: (<FormattedMessage id="stripes-acq-components.shortcut.paste" />),
+    shortcut: 'mod+v',
+  },
+  {
+    name: 'find',
+    label: (<FormattedMessage id="stripes-acq-components.shortcut.find" />),
+    shortcut: 'mod+f',
+  },
+  {
+    name: 'openShortcutModal',
+    label: (<FormattedMessage id="stripes-acq-components.shortcut.find" />),
+    shortcut: 'mod+alt+k',
+  },
+];
```

Comment:
```suggestion
export const acqCommands = [
  {
    name: 'new',
    shortcut: 'alt+n',
  },
  {
    name: 'edit',
    shortcut: 'mod+alt+e',
  },
  {
    name: 'save',
    shortcut: 'mod+s',
  },
  {
    name: 'expandCollapse',
    shortcut: 'spacebar',
  },
  {
    name: 'expandAllSections',
    shortcut: 'mod+alt+b',
  },
  {
    name: 'collapseAllSections',
    shortcut: 'mod+alt+g',
  },
  {
    name: 'search',
    shortcut: 'mod+alt+h',
  },
  {
    name: 'close',
    shortcut: 'esc',
  },
  {
    name: 'copy',
    shortcut: 'mod+c',
  },
  {
    name: 'cut',
    shortcut: 'mod+x',
  },
  {
    name: 'paste',
    shortcut: 'mod+v',
  },
  {
    name: 'find',
    shortcut: 'mod+f',
  },
  {
    name: 'openShortcutModal',
    shortcut: 'mod+alt+k',
  },
]
.map(({ name, shortcut }) => ({
  name,
  label: <FormattedMessage id=`stripes-acq-components.shortcut.${name}` />,
  shortcut,
}));
```


That may be a way to please Sonar.

You got to adjust translation names to match the `name`

Reply:
Or even define it like that:
```js
export const commandsMap = {
  new: 'alt+n',
  edit: 'mod+alt+e',
  // ...
};
```
and then process it into needed array.
That would be much cleaner, and more compact too.

Reply:
not sure that we need to do smth to avoid Sonar issues with duplications in this case. @NikitaSedyx what do you think?

Reply:
couldn't hurt to try )

Reply:
I like option with `commandsMap`, but not sure it will avoid sonar duplication